### ISSUE TITLE
segtrain errors out when run with cosine schedule; works fine without it

### ISSUE THREAD

**Issue by rohanchn** [mittagessen/kraken#491]
Hi,

I ran the command ` ketos segtrain -f alto -d cuda:0 -N 70 --schedule cosine --resize both -i {path_to_i-model} -o {path_to_o-model} -t xml.xmllist --logger tensorboard -tl` and got a `TypeError` : `lr_scheduler_step() missing 1 required positional argument: 'metric'`

Attaching the traceback as [error.txt](https://github.com/mittagessen/kraken/files/11241378/error.txt).

It works fine when I run the command without `--schedule cosine`

I think I am missing something?

**Comment by rohanchn** [comment]
This is with `kraken==4.3.9`

**Comment by mittagessen** [comment]
It's already been fixed in a11989d2e568967c0934921c7c81b3a47395e4a9.

**Comment by rohanchn** [comment]
Thanks!
